dts/bindings/display/galaxycore,gc9x01x.yaml

```diff
@@ -44,6 +44,7 @@ properties:
 
   display-inversion:
     type: boolean
+    default: false
```

Comment:
This is not needed for boolean properties. If they are absent they are false by default

Reply:
It's actually even disallowed :) (hence some of CI failures)